DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -75,6 +76,19 @@ function PreFirstEntryMapFixes()
         break;
     case "02_NYC_HOTEL":
         Spawn(class'#var(prefix)Binoculars',,, vectm(-610.374573,-3221.998779,94.160065)); //Paul's bedside table
+
+        //Restore answering machine message, but in mission 2 (conversation is in mission 3);
+        //tad=Spawn(class'#var(prefix)TAD',,, vectm(-290,-2380,115),rotm(0,0,0));
+        //tad.BindName="AnsweringMachine";
+        //CreateAnsweringMachineConversation(tad);
```

Comment:
can we clean up these comments? seems like nothing is calling CreateAnsweringMachineConversation? it should be called from AnyEntry I think since conversations are transient